rules/okta_rules/okta_password_accessed.py

```diff
@@ -0,0 +1,39 @@
+from panther_base_helpers import deep_get, lst_iter
+
+
+def rule(event):
+    global target_user
+    global target_app_names
+    # event['target'] = [{...}, {...}, {...}]
+    target_user = lst_iter(
+        event.get("target", [{}]), 'alternateId', 'Type', 'AppUser')
+    target_app_names = target_app_names = lst_iter(
+        event.get("target", [{}]), 'alternateId', 'Type', 'AppInstance')
+
+    if (
```

Comment:
filter block should move up to line 8 before iteration/deep inspection. We can avoid the work of iterating on things that will never pass the filter.